op-devstack/sysgo/system_synctester_ext.go

```diff
@@ -0,0 +1,94 @@
+package sysgo
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/devkeys"
+	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-node/chaincfg"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+type DefaultMinimalExternalELSystemIDs struct {
+	L1   stack.L1NetworkID
+	L1EL stack.L1ELNodeID
+	L1CL stack.L1CLNodeID
+
+	L2   stack.L2NetworkID
+	L2CL stack.L2CLNodeID
+	L2EL stack.L2ELNodeID
+
+	SyncTester stack.SyncTesterID
+}
+
+func NewDefaultMinimalExternalELSystemIDs(l1ID, l2ID eth.ChainID) DefaultMinimalExternalELSystemIDs {
+	ids := DefaultMinimalExternalELSystemIDs{
+		L1:         stack.L1NetworkID(l1ID),
+		L1EL:       stack.NewL1ELNodeID("l1", l1ID),
+		L1CL:       stack.NewL1CLNodeID("l1", l1ID),
+		L2:         stack.L2NetworkID(l2ID),
+		L2CL:       stack.NewL2CLNodeID("sequencer", l2ID),
```

Comment:
L2CL behaves as a verifier.
```suggestion
		L2CL:       stack.NewL2CLNodeID("verifier", l2ID),
```